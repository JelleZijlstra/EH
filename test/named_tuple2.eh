printvar (3, 4, foo: 'bar')
