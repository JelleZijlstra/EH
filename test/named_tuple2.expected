%%stdout%%
@extended tuple <2> [
	@int 3
	@int 4
	"foo": @string "bar"
]
%%stderr%%
